pkgs/development/libraries/libjpeg-turbo/default.nix

```diff
@@ -1,37 +1,29 @@
-{ stdenv, fetchurl, fetchpatch, cmake, nasm, enableStatic ? false, enableShared ? true }:
+{ stdenv, fetchFromGitHub, cmake, nasm, enableStatic ? false, enableShared ? true }:
 
 stdenv.mkDerivation rec {
 
   pname = "libjpeg-turbo";
   version = "2.0.4";
 
-  src = fetchurl {
-    url = "mirror://sourceforge/${pname}/${pname}-${version}.tar.gz";
-    sha256 = "01ill8bgjyk582wipx7sh7gj2nidylpbzvwhx0wkcm6mxx3qbp9k";
+  src = fetchFromGitHub {
+	  owner = "libjpeg-turbo";
+	  repo = "libjpeg-turbo";
+	  rev = "2.0.5";
```

Comment:
Can use `version` here

Reply:
Do you want me to put libjpeg-turbo: 2.0.4 -> 2.0.5 in the commit header too? Or the commity message body?

Reply:
Whatever happens, it's got to match `version` a few lines above.

Where possible people generally do one commit per package in PRs. But this only makes sense when you can put them in an order that doesn't leave anything broken in its in-between states. I think you can probably do that if you put the `libtiff` and `libjpeg-turbo` changes first...